source/introduction.md

```diff
@@ -124,6 +126,28 @@ to the following:
 - A name cannot start with a number
 - A name can only contain alpha-numeric characters and underscores (A-Z, a-z, 0-9, and _ )
 
+%---------------------------------------------------------------------------------------------------
+(s:name.matching)=
+## Lattice Element Name Matching
+
+Lattice element name matching is the process of finding the set of lattice elements that 
+are matched to a given string. Name matching is important in a number of instances including
+lattice expansion where there are [`Fork`](#s:forking) elements and for evaluating mathematical
+expressions.
+
+The simplist form of name matching is if the string matches
+the `name` field of an element or elements. Regular expressions can be used. 
+Regular expressions must conform to the PCRE2 standard. 
```

Comment:
Add link to the PCRE2 standard (e.g., https://www.pcre.org/)?

Reply:
Done.